src/main/features/directions-questionnaire/views/hearing-location.njk

```diff
@@ -29,8 +30,19 @@
             {% elseif party === MadeBy.CLAIMANT %}
               <p>{{ t('You can request any location - {{ courtName }} is the nearest to your home address you gave us.', { courtName: courtName }) }}
             {% endif %}
-            <h1
-              class="heading-medium">{{ t('Do you want to choose {{ courtName }}?', { courtName: form.valueFor('courtName') }) }}</h1>
+
+            <details class="form-group analytics-click-event-trigger" data-event-label="Response: Find out what each response means">
```

Comment:
Label has been copied - should be updated

Reply:
Done